run-platform.sh

```diff
@@ -190,6 +190,14 @@ setup_env() {
   DEFAULT_AUTH_KEY="unstract"
 
   for service in "${services[@]}"; do
+    # Skip services that are spawned at runtime
+    for ignore_service in "${spawned_services[@]}"; do
+      if [[ "$service" == "$ignore_service" ]]; then
+        echo -e "Skipped env for ${blue_text}$service${default_text} as it's spawned at runtime"
```

Comment:
@chandrasekharan-zipstack 
```suggestion
        echo -e "Skipped env for ${blue_text}$service${default_text} as it's generated at runtime"
```